edgexfoundry/edgex-examples
# refactor: Upgrade cloud event

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:


## Testing Instructions
run `go test ./pkg/transformations`
start up edgex with no-secty
run `make build`
run `./app-serve -cp -dev`
verify service starts up
